app/src/main/java/net/skyscanner/backpack/demo/compose/ChipGroupStory.kt

```diff
@@ -0,0 +1,144 @@
+import net.skyscanner.backpack.compose.chipgroup.single.BpkSingleChipItem
+import net.skyscanner.backpack.compose.chipgroup.BpkSingleSelectChipGroup
+import net.skyscanner.backpack.compose.icon.BpkIcon
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.compose.tokens.Deals
+import net.skyscanner.backpack.compose.tokens.Heart
+import net.skyscanner.backpack.demo.R
+import net.skyscanner.backpack.demo.components.ChipGroupComponent
+import net.skyscanner.backpack.demo.meta.ComposeStory
+
+@Composable
+@ChipGroupComponent
+@ComposeStory("Single Select Rail")
+fun SingleSelectChipGroupStoryRail(modifier: Modifier = Modifier) =
+    SingleSelectChipGroupDemo(BpkSingleChipGroupType.RAIL, modifier)
+
+@Composable
+@ChipGroupComponent
+@ComposeStory("Single Select Wrap")
+fun SingleSelectChipGroupStoryWrap(modifier: Modifier = Modifier) =
+    SingleSelectChipGroupDemo(BpkSingleChipGroupType.WRAP, modifier)
+
+@Composable
+private fun SingleSelectChipGroupDemo(type: BpkSingleChipGroupType, modifier: Modifier = Modifier) {
+    Column(modifier) {
+        Box(modifier = Modifier.weight(1f)) {
+            Image(
+                painter = painterResource(R.drawable.city),
+                contentDescription = null,
+                contentScale = ContentScale.Crop,
+                modifier = Modifier.fillMaxSize(),
+            )
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(BpkSpacing.Base)
+                    .background(Color.Transparent),
+                verticalArrangement = Arrangement.spacedBy(BpkSpacing.Base),
+            ) {
+
+                SingleSelectChipGroupSample(type = type, style = BpkChipStyle.OnImage)
+            }
+        }
+        Box(modifier = Modifier.weight(1f)) {
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .background(BpkTheme.colors.surfaceContrast)
+                    .padding(BpkSpacing.Base),
+                verticalArrangement = Arrangement.spacedBy(BpkSpacing.Base),
+            ) {
+
+                SingleSelectChipGroupSample(type = type, style = BpkChipStyle.OnDark)
+            }
+        }
+        Box(modifier = Modifier.weight(1f)) {
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .background(Color.Transparent)
+                    .padding(BpkSpacing.Base),
+                verticalArrangement = Arrangement.spacedBy(BpkSpacing.Base),
+            ) {
+
+                SingleSelectChipGroupSample(type = type, style = BpkChipStyle.Default)
+            }
+        }
+    }
+}
+
+@Composable
+internal fun SingleSelectChipGroupSample(
+    type: BpkSingleChipGroupType,
+    modifier: Modifier = Modifier,
+    defaultIndex: Int = -1,
```

Comment:
I commented on this in the previous version, but I think it got lost. Let's remove the `-1` case (and non-selected tests) as it isn't really a supported case to _not_ have a selected item.